### ISSUE TITLE
Wrong `to_gregorian()` output

### ISSUE THREAD

**Issue by beekter** [seanredmond/juliandate#4]
Hi! Am I doing something wrong?

```
>>> import juliandate as jd
>>> jd.to_gregorian(2440646.5)
(1970, 2, 29, 0, 0, 0, 0)
```

which apparently must be March 1st 
```
The calendar date for 2440646.5 is 0:0:0.00 UT on March 1, 1970 .
```
according to https://www.aavso.org/jd-calculator

**Comment by seanredmond** [comment]
Maybe a rounding error somewhere when determining whether it's a leap year 

    >>> jd.to_gregorian(2440646.49)
    (1970, 2, 28, 23, 45, 36, 19)
    >>> jd.to_gregorian(2440646.5)
    (1970, 2, 29, 0, 0, 0, 0)
    >>> jd.to_gregorian(2440646.749)
    (1970, 2, 29, 5, 58, 33, 599986)
    >>> jd.to_gregorian(2440646.75)
    (1970, 3, 1, 6, 0, 0, 0)

Looking into it

**Comment by seanredmond** [comment]
Released new version 1.0.4 that should fix this.

Thanks for the report!
